### PR TITLE
Tweaks to RN based on feedback

### PR DESCRIPTION
**Pull request by nadinequinr3**
Preview currently here: https://nadine.preview.docs.r3.com/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.html

### CODE REVIEW

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ table th:nth-of-type(2) {
...
 
-| Topic  | Specific Change | 
+| Domain  | Specific Change | 
 | -------------------------------------------- | -------------------------------------------------- |
 | **CorDapp Development**                      | **Flows with Transient Errors** - Flows did not correctly handle transient errors while sessions were in progress. In some cases, the 'error' flag was cleared early by other events, incorrectly indicating that the error had been resolved. This flag can now only be cleared when the event that caused the transient error is retried.  |
 
 ## Known Limitations and Issues
 
-There is no support for upgrades from the early access beta versions.
+There is no support for upgrades from the beta versions.
```
Comment (fenryka) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md:51]: what does this mean? I think we should be saying "During the Beta process we do not guarantee the stability of our user APIs, thus seemless upgrade between Beta versions is not supported. 

After all, we know that its possiblem, only that we don't garuntee it

**Comment by fenryka** [comment]
We need to chase up the team on whats supposed to be Fixed

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md]: Updated. Thanks.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by colin-hillis — APPROVED** [review]
Approved so that we can get other changes made. The additional RNs will come later, once there is agreement on engineering side about scope.

**Comment by nadinequinr3** [comment]
> We need to chase up the team on whats supposed to be Fixed

@fenryka While the bugs discussion is ongoing, can we approve and merge this PR? We can address any additional bugs in another PR.

**Pull request merged**